dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/AbstractClientStream.java

```diff
@@ -155,7 +160,11 @@ protected Metadata createRequestMeta(RpcInvocation inv) {
                         (String) inv.getObjectAttachments().remove(CommonConstants.APPLICATION_KEY))
                 .putIfNotNull(TripleHeaderEnum.CONSUMER_APP_NAME_KEY.getHeader(),
                         (String) inv.getObjectAttachments().remove(CommonConstants.REMOTE_APPLICATION_KEY))
-                .putIfNotNull(TripleHeaderEnum.SERVICE_GROUP.getHeader(), inv.getInvoker().getUrl().getGroup());
+                .putIfNotNull(TripleHeaderEnum.SERVICE_GROUP.getHeader(), inv.getInvoker().getUrl().getGroup())
+                .putIfNotNull(TripleHeaderEnum.GRPC_ENCODING.getHeader(),
+                    ConfigurationUtils.getGlobalConfiguration(ApplicationModel.defaultModel()).getString(Constants.COMPRESSOR_KEY, DEFAULT_COMPRESSOR))
```

Comment:
plz Obtain the configuration in multi-instance mode

Reply:
done